debezium-testing/debezium-testing-system/src/test/java/io/debezium/testing/system/tests/mongodb/OcpMongoTests.java

```diff
@@ -63,13 +63,12 @@ public void shouldStreamInReplicaSetMode(OcpMongoShardedController dbController)
                 .put("topic.prefix", connectorName);
         connectController.deployConnector(connectorConfig);
         String topic = connectorName + ".inventory.customers";
+        assertions.assertTopicsExist(
+                connectorName + ".inventory.customers");
 
         insertCustomer(dbController, "Eve", "Sharded", "eshard@test.com");
 
-        assertions.assertTopicsExist(
-                connectorName + ".inventory.customers");
         awaitAssert(() -> assertions.assertRecordsContain(topic, "eshard@test.com"));
-        awaitAssert(() -> assertions.assertMinimalRecordsCount(topic, 5));
```

Comment:
I think you can assert exact records count here. But I may be wrong

Reply:
The thing is I delete a customer before this. This signals kafka, that it can clear all records related to this customer, so just in case I use the minimalRecordCount